plugwise_usb/nodes/circle.py

```diff
@@ -474,6 +474,7 @@ async def get_missing_energy_logs(self) -> None:
             missing_addresses = sorted(missing_addresses, reverse=True)
             for address in missing_addresses:
                 await self.energy_log_update(address)
+                await sleep(0.1)  # Test with a small delay
```

Comment:
The sleep should not be necessary at all. Adding it masks the real issue at the submit queue.
Any "communication overload" should be handled properly at the submit queue so we are able to do all the energy requests in a single `gather` request.